CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
 * Adds support for Azure Virtual Machine Scale Sets in the Azure plugin
   ([PR](https://github.com/hashicorp/boundary-plugin-azure/pull/22)).
 
+### Bug fixes
+
+* Fixed an issue in the worker where closing a channel failed to exit a loop,
```

Comment:
I would add clarification on what a channel is in this context
```suggestion
* Fixed an issue in the worker where closing an SSH channel failed to exit a loop,
```

Reply:
I added this clarification to the changelog item, thanks for pointing this out.